sunpy/coordinates/transformations.py

```diff
@@ -197,52 +213,64 @@ def hpc_to_hpc(heliopcoord, heliopframe):
     return hpc
 
 
-# Make a transformation graph for the documentation, borrowed lovingly from
-# Astropy.
+def _make_rotation_matrix(start_representation, end_representation):
```

Comment:
Also, it might make sense to make a PR to put this in Astropy?  I'm sure it'd be useful in some non-solar contexts... If so, I think this could be added to `astropy.coordinates.matrix_utilities`, and you could change this function to do something like:
```
def _make_rotation_matrix(start_representation, end_representation):
    if astropy.version.major > 2 and astropy.version.minor > 0:
        from astropy.coordinates.matrix_utilities import make_rotation_matrix_from_reprs
        return make_rotation_matrix_from_reprs(start_representation, end_representation)

    ... the existing function goes here ...
```